.gitignore

```diff
@@ -1 +1,2 @@
 build/
+dcos-commons-tools/
```

Comment:
The tools are being fetched/unpacked from a .tgz in `build.sh`: https://github.com/mesosphere/spark-build/pull/58/files/7697bceae7b4311dc56b7c0816a63f8c66a5de8a#diff-d296a94261ea80f5a08b1cb3f93fb777R29